lib/customer.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'menu.rb'
+
+class Customer
+
+  attr_reader :order, :menu
+
+  def initialize(menu = Menu.new)
+    @order = {}
+    @menu = menu.dishes
+  end
+
+  def display_menu
+   menu.each { |key, value| "#{key}, £#{value}" }
+  end
+
+  def place_order(dish, quantity)
+    fail "This dish is not on the menu" unless menu.has_key?(dish)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.